rough_draft_aim2.md

```diff
@@ -0,0 +1,17 @@
+
+Human Cell Atlas' partnership with the Immunological Genome Project (immgenH) will result in gene expression-based immunocyte phenotyping at unprecedented resolution, making it well-suited for the problem of studying rheumatic/autoimmune disorders and for addressing the question of the interplay between tissue-resident immune cells and those cells infiltrating from peripheral blood in these disease states.
+
+We will apply Pathway-level Information ExtractoR (**PLIER**)([Mao ref](http://dx.doi.org/10.1101/116061)), which has been demonstrated to be effective in extracting cell type-specific signals from bulk gene expression data, to bulk autoimmune/rheumatic disease data. PLIER requires bulk expression data and prior knowledge as input. In this case, 
+we will encorporate cell subset and activation state information obtained from single-cell data as prior information (e.g., sets of genes that distinguish cell-types). (Note that this analysis framework is agnostic to the particular methodology used to generate low-dimensional representations of cell-types/states.) 
+
+### Evaluations
+
+We must evaluate PLIER's performance in addressing the subtle question of immune lineage phenotype and propose to do so in the following ways:
+
+1. **Simulation experiments** - We will evaluate PLIER for use in solid organs by generating simulated bulk data drawn from purified cell lineages, including non-immune lineages where available. We will combine purified cell expression data at different proportion levels and use single-cell-based cell type marker sets as prior information. This tests the hypothesis that cell type proportions can be estimated in heterogeneous, bulk solid organ data using PLIER and low-dimensional representations of single-cell data. Utilizing bulk pure cell expression from immune cell transcriptome projects (e.g., macrophage ([Xue ref](http://dx.doi.org/10.1016/j.immuni.2014.01.006))) that contain perturbations with many stimuli or stages of differentiation in our simulations will allow us to evaluate performance when highsimilar cell type states are added at different proportions.
```

Comment:
> single-cell-based cell type marker sets

~Do you have a specific reference in mind?~ _edit_ i see now the macrophage example